data/meta/tokens/1/0x79e281bc69a03dabccd66858c65ef6724e50aebe.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve sdYFI-YFI Pool",
+  "symbol": "sdYFI",
+  "description": "This token represents a Curve liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This pool contains YFI and sdYFI, a Staked YFI token from StakeDAO.",
```

Comment:
This pool contains YFI and sdYFI, a tokenized veYFI from StakeDAO.

Reply:
fixed